### ISSUE TITLE
Adding repositories doesn't seem to work

### ISSUE THREAD

**Issue by olivierlacan** [dependency-db/support#3]
![image](https://cloud.githubusercontent.com/assets/65950/14019048/266087ec-f1d1-11e5-8752-de7435acf47a.png)

Clicking the add button apparently sends an XHR request but even seconds later nothing happens and on refresh the Add Repository button re-appears.

Is it hooked up or is there a service/API issue?

**Comment by boffbowsh** [comment]
Hi, thanks for the report. I'm not sure why that happened. I don't see any errors for that repo. When you refreshed, did you try clicking the Add Repository button again? I'll have a further look through the logs later, but for now I've added that repo manually.

**Comment by olivierlacan** [comment]
I did try again. Nothing happened the second time either.

On March 24, 2016 at 3:31:23 PM, Paul Bowsher (notifications@github.com) wrote:

Hi, thanks for the report. I'm not sure why that happened. I don't see any errors for that repo. When you refreshed, did you try clicking the Add Repository button again? I'll have a further look through the logs later, but for now I've added that repo manually.
